test/operations/manager.go

```diff
@@ -182,7 +182,7 @@ func (m *Manager) SendL2Deposit(ctx context.Context, tokenAddr common.Address, a
 		return err
 	}
 
-	auth.GasPrice = big.NewInt(0) // TODO remove when the executor is fixed to estimate gas
+	auth.GasPrice = big.NewInt(0) // TODO set the appropriate value
```

Comment:
If you don't set the value, It should be caulculated automatically

Reply:
fair enough!